discretize/MeshIO.py

```diff
@@ -118,7 +118,7 @@ def unpackdx(fid, nrows):
         return tensMsh
 
     @classmethod
-    def readUBC(TensorMesh, fileName, meshdim=None):
+    def readUBC(TensorMesh, fileName, meshdim=None, folder=''):
```

Comment:
do we want `folder` or `directory` ? 
cc @rowanc1: do you know what is more common?

Reply:
Indeed directory does sound better.